tasks/main.yml

```diff
@@ -17,12 +17,11 @@
     - sshd
 
 - name: Installed
-  action: >
-    {{ ansible_pkg_mgr }}
-    name="{{ item }}"
-    state=installed
-  with_items: "{{ sshd_packages }}"
-  when: ansible_pkg_mgr != 'unknown'
+  package:
+    name: "{{ sshd_packages }}"
+    state: installed
+    update_cache: yes
```

Comment:
are these defaults true everywhere and do we want them to trump defaults set in our ansible configs?